cms/private_media/tests/test_documents.py

```diff
@@ -301,21 +301,25 @@ def test_private_document(self):
         response = self.client.get(self.private_document.url)
         self.assertEqual(response.status_code, 200)
 
-    def test_public_document(self):
+    @override_settings(IS_EXTERNAL_ENV=True)
+    def test_serve_private_document_external_env(self):
+        """Test the serve view behaviour for private documents in an external environment."""
+        self.test_serve_private_document()
```

Comment:
This feels like a really strange way of handling it. It might be better to use `subTest` to run the test twice.

Reply:
"really strange" is a bit strong, no? To me, this makes the intention more obvious.

Reply:
@RealOrangeOne Could you mark this as resolved please?